web/modules/custom/ys_base/src/Plugin/DeployStep/CreateContentModelDeployStep.php

```
@@ -0,0 +1,75 @@
+
+    return $this->configFactory->get('node.type.page')->get('type') !== NULL ? 'content model already exists' : NULL;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function run(): void {
+    // The recipe lives at the project root, but deploy steps run with the
+    // Drupal root as the working directory, so it is resolved against the
+    // parent of the Drupal root.
+    $recipe = Recipe::createFromDirectory(dirname($this->appRoot) . '/recipes/page');
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\Core\Recipe\Recipe' in method 'run'. (undefined)

(StaticAccess)